src/services/playerEvents.ts

```diff
@@ -98,12 +100,20 @@ const syncNowPlayingItemWithTrack = async () => {
   setTimeout(sync, 1000)
 }
 
+const handleQueueEnded = async () => {
+  setTimeout(async () => {
+    await setPlaybackPosition(0)
+    hideMiniPlayer()
+  }, 0)
+}
+
 module.exports = async () => {
   PVTrackPlayer.addEventListener('playback-error', (x) => console.log('playback error', x))
 
   PVTrackPlayer.addEventListener('playback-queue-ended', async (x) => {
     console.log('playback-queue-ended', x)
     syncNowPlayingItemWithTrack()
```

Comment:
Does this need to be called with await? so that handleQueue ended happens AFTER this is done? Or is this not asynchronous?